crates/subspace-runtime/src/lib.rs

```diff
@@ -319,6 +319,7 @@ parameter_types! {
     // Disable solution range adjustment at the start of chain.
     // Root origin must enable later
     pub const ShouldAdjustSolutionRange: bool = false;
+    pub const BlockSlotCount: u32 = 6;
```

Comment:
This should be dependent on consensus `SLOT_PROBABILITY`, not hardcoded.

Reply:
The name `BlockSlotCount` may be misleading but it is the count for the consensus block not for the slot.